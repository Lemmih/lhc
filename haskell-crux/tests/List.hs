id x = x

nul = id []

