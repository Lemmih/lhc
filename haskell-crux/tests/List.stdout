Main.id : ∀ v2. v2 →  v2
Main.id =
  λ @.v2^1:* Main.x^2:v2 → 
  Main.x^2
Main.nul : ∀ v5. [v5]
Main.nul =
  λ @.v5^3:* → 
  Main.id (@.LHC.Prim.List @.v5^3)
    LHC.Prim.Nil
